tests/unit/installer/test_installation_manager.py

```diff
@@ -63,6 +63,15 @@ def test_corrupt_config(mocker):
     assert len(user_installations) == 0
 
 
+def test_corrupt_config_yaml(mocker):
+    ws = mocker.patch("databricks.sdk.WorkspaceClient.__init__")
+    ws.users.list.return_value = [User(user_name="foo")]
+    ws.workspace.download.return_value = io.StringIO("version: 2\ntacl: extra colon: test")
+    installation_manager = InstallationManager(ws)
```

Comment:
Do the test with corrupt yaml

Reply:
yes i have created a string with corrupt yaml having extra ":" re-wrote the test cases to showing raises right Exception